runbotics/runbotics-desktop/src/action/microsoft/excel/excel.service.ts

```diff
@@ -221,7 +233,9 @@ export class ExcelService {
             );
 
         if (!drive) {
-            throw new Error(`Site ${sessionInfo.siteName} does not contain ${sessionInfo.listName} list`);
+            throw new Error(`
+                Site ${sessionInfo.siteName} does not contain ${sessionInfo.listName} or ${sessionInfo.listName} is not of type 'Document library'`
```

Comment:
adjust it accordingly:
`Site ${sessionInfo.siteName} does not contain "${sessionInfo.listName}" list or "${sessionInfo.listName}" is not a list of type 'Document library'`